locale/pt_PT/locale.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```

Comment:
I think this needs another `../` -- check the `en_US` equivalent for an example.